### PR TITLE
insert statistics image in infobox of node

### PR DESCRIPTION
**Pull request by Little-Ben**
This pull request inserts statistics image in infobox - created by backend option --with-rrd.
You can use the config options to either disable showing the statistics 
or only show a textual link in infobox. 
URL can be absolute or relativ.

### CODE REVIEW

**Comment by Little-Ben** [comment]
corrections according tcatm's suggestions
- remove dictGen as it is not necessary for d.nodeinfo.node_id,
- image width via css,
- removing empty lines,
- config as array "nodeInfos"

**Comment by Little-Ben** [comment]
- added array for noedInfos: possibility to have multiple definitions
- changed thumbnail / caption
- add height and max-height to img css

**Comment by jplitza** [comment]
I'd like to advocate for client-side graph drawing, for example (but not necessarily) using [javascriptrrd](http://javascriptrrd.sourceforge.net/), which could read the source RRD files directly. This enables a more responsive output, with tooltips showing precise values etc. Unresponsive images aren't really the style of the meshviewer.

File: lib/infobox/node.js [hunk 1]
```diff
@@ -133,6 +133,46 @@ define(["moment", "numeral", "tablesort", "tablesort.numeric"],
...
+        //flag useTextLink=true (default), to use text link, since no thumbnail available
+      }
+
+      if (nodeInfosElement.type === "link" || useTextLink === true){
+        //show only text link
+        var statCaption = "Bild"
+        if (nodeInfosElement.caption !== undefined)
+          statCaption = nodeInfosElement.caption
+        var statText = document.createTextNode(statCaption.replace("{NODE_ID}", d.nodeinfo.node_id))
+        statLnk.appendChild(statText)
```
Comment (tcatm) [inline lib/infobox/node.js]: `statLnk.textContent = statCaption.replace...` wäre kürzer

**Comment by Little-Ben** [comment]
Changes:
- removed useTextLink
- using textContent to shorten code for text links

javascriptrrd:
as tcatm said in IRC javascriptrrd will not be used as this is planned to be rendered with influxdb in "near future"

this PR is only as work around until influxdb is used

**Comment by nemesisdesign** [comment]
heya, if you want to collaborate on building some influxdb tool in a separate library i'd be interested.

I'm actively following the development of influxdb and the python-influxdb client:
https://github.com/influxdb/influxdb-python/commits?author=nemesisdesign

Federico (Ninux.org)

**Pull request closed without merging**